installers/olm/README.md

```diff
@@ -0,0 +1,6 @@
+1. Set VERSION env variable
+2. Update rbac and operator in config
+2. Choose the mode (cluster or namespace) and update bundle/kustomization.yaml and makefile with necessary mode.
+3. update description in bundle.csv.yaml
+4. Update variables in makefile.
+5. ```make bundles```
```

Comment:
I get Stop when running `make`. Wha should be provided here?
` % make bundles
make: *** No rule to make target `bundles'.  Stop.
`

Reply:
I have run it from wrong folder. Can we add here from where we need to run it?